src/index.html

```diff
@@ -5,8 +5,366 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img 
+          src="images/logo.png" 
+          alt="logo" 
+          class="header__logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__wrap">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#Apple"> Apple </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#Samsung"> Samsung </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#Smartphones"> Smartphones </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              class="nav__link" 
+              data-qa="nav-hover"
+              href="#Laptops_&_Computers"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#Gadgets"> Gadgets </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#Tablets"> Tablets </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#Photo"> Photo </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#Video"> Video </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="catalog main__catalog">
+        <div class="card" data-qa="card">
+          <img 
+            class="card__img" 
+            src="./images/imac.jpeg" 
+            alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA&sol;A)"
+          >
+    
+          <h1 class="card__title">APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)</h1>
+    
+          <p class="card__code">Product code: 195434</p>
+    
+          <div class="card__revievs">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+    
+            <p class="card__revievs-title">Reviews: 5</p>
+          </div>
+    
+          <div class="card__price">
+            <p class="card__price-title">Price:</p>
+            <h2 class="card__price-sum">$2,199</h2>
```

Comment:
it looks like a simple text
```suggestion
            <p class="card__price-sum">$2,199</p>
```